deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,86 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+/*Schrijf hier jouw css*/
+
+body {
+    background-color: #F2F3D9;
+    margin: 0;
+}
+
+.navbar {
+    background-color: #030027;
+    color: #fff;
+    padding: 40px;
+}
+
+.navbar .container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navbar a {
+    color: #fff;
+    text-decoration: none;
+    font-family: "Roboto Light", sans-serif;
+    font-size: 18px;
+}
+
+.logo {
+    font-family: "Roboto Light", sans-serif;
+    font-size: 18px;
+    font-weight: bold;
+}
+
+.navbar ul {
+    display: flex;
+    gap: 40px;
+}
```

Comment:
grow en shrink ontbreken, en in het nakijk formulier geven ze aan de UL n og een justify-content: space-between; mee